### PR TITLE
Show oc login command with proxyURL in Landing Page

### PR DESCRIPTION
**Pull request by ranakan19**
Fixes: https://issues.redhat.com/browse/CRT-1396
related e2e test - https://github.com/codeready-toolchain/toolchain-e2e/pull/440

### CODE REVIEW

File: pkg/signup/service/signup_service.go [hunk 1]
```diff
@@ -288,6 +288,7 @@ func (s *ServiceImpl) GetSignup(userID string) (*signup.Signup, error) {
 		if err != nil {
 			return nil, errors2.Wrapf(err, "error when retrieving ToolchainStatus to set Che Dashboard for completed UserSignup %s", userSignup.GetName())
 		}
+		signupResponse.ProxyURL = status.Status.HostRoutes.ProxyURL
```
Comment (alexeykazakov) [inline pkg/signup/service/signup_service.go:291]: Let's add it to our unit tests. And I think we could/should even verify that the proxyURL is set in our e2e tests.

**Review by alexeykazakov — COMMENTED** [review]
Can you add some screenshots?

**Comment by ranakan19** [comment]
D114524E-7B94-426D-A3D2-CF3A97A0D7E5

**Comment by ranakan19** [comment]
154DEA90-5D69-4D39-A667-48385F348939
After clicking 'Access Login Command'

**Comment by alexeykazakov** [comment]
The screenshots look good! But let's rename it to "Proxy login command" or something like this. So it's clear that it's about the proxy.

**Review by alexeykazakov — APPROVED** [review]
Looks good!

**Review by MatousJobanek — APPROVED** [review]
This looks good :+1: Good job @ranakan19 :rocket:

**Pull request merged**